quartz/build.ts

```diff
@@ -60,7 +60,7 @@ async function buildQuartz(argv: Argv, mut: Mutex, clientRefresh: () => void) {
 
   const release = await mut.acquire()
   perf.addEvent("clean")
-  await rimraf(output)
```

Comment:
this should use `path.join` to avoid cases where `output` already has a trailing slash